index.html

```diff
@@ -52,7 +52,113 @@ <h1>  I’m Lalla <br>
+                        <li><a href="#" class="programming">html</a></li>
+                        <li><a href="#" class="programming">css</a></li>
+                        <li><a href="#" class="programming">javascript</a></li>
+                    </ul>
+
+                    <button type="button">See Project</button>
+                </div>
+            </div>
+            <div class="job">
+                <img src="image/SnapshootPortfolio5.jpg" id="ICO">
+                <div class="content">
+                    <h3> Multi-Post Stories</h3>
+                    <ul>
+                        <li>Canopy</li>
+                        <li>Back End DEv</li>
+                        <li>2015</li>
+                    </ul>
+                    <p class="Descriptions">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul>
+                        <li><a href="#" class="programming">html</a></li>
+                        <li><a href="#" class="programming" >css</a></li>
+                        <li><a href="#" class="programming">javascript</a></li>
+                    </ul>
+                    <button type="button">See Project</button>
+                </div>
+            </div>
+            <section class="section-1">
+              <div class="sct-1">
+                  <h2>About  Myself</h2>
+                  <p class ="Descriptions"> Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                  <a href="#" class="connect">LET'S CONNECT</a><br>
+                      <div class="contacts">
+                          <ul><br>
+                              <li><a href="https://github.com/leloufadel" class ="link"><i class="fa fa-github fa-2x"></i></a></li>
+                              <li><a href="https://www.linkedin.com/in/el-vadel-lalaaicha-8563a6202" class ="link"><i class="fa fa-linkedin fa-2x"></i></a></li>
+                              <li><a href="#" class ="link"><i class="fa fa-hand-peace-o"></i></a></li>
+                              <li><a href="#" class ="https://twitter.com/leloufadel"><i class="fa fa-twitter"></i></a>
+                              <li><a href="" class = "link"><i class="fab fa-medium-m"></i></a></li>
+                          </ul>
+                      </div>
+                      <button type="button">Get Resume</button>
+              </div>
+              <div class="skills">
+                <p class="skill"> Languages</p>
+                <ul class = "languages">
+                    <li><img src="image/js.jpg" alt=""><a href="#" class = "tech" >JavaScript</a></li>
+                    <li> <img src="image/html.jpg" alt=""><a href="#" class = "tech" >HTML5</a></li>
+                    <li><img src="image/css.jpg" alt=""><a href="#" class = "tech">CSS</a></li>
```

Comment:
- [ ] According to the project's requirements, the categories eg "Languages", "Frameworks" etc should be list item `<li>` as well, please make sure to implement that.